.github/workflows/main.yml

```diff
@@ -432,6 +432,19 @@ jobs:
         run: |
           RUST_TEST_THREADS=1 cargo test --manifest-path="libvcx/Cargo.toml" -F "pool_tests";
 
+  test-integration-resolver:
```

Comment:
The name is not quite accurate right now, the command also currently run unit tests too. Either update now, or tweak the cargo cmd